frontend-implicaction/src/app/core/services/api-endpoints.service.ts

```diff
@@ -224,6 +224,23 @@ export class ApiEndpointsService {
       });
   }
 
+  getAllActiveJobEndpoint(pageable: Pageable, criteria: JobCriteriaFilter, archive: any) {
```

Comment:
```suggestion
  getAllActiveJobEndpoint(pageable: Pageable, criteria: JobCriteriaFilter, archive: any): string {
```
devrait corriger ton problème de ci